compiler/src/main/scala/org/scalajs/core/compiler/PrepJSInterop.scala

```diff
@@ -693,7 +694,7 @@ abstract class PrepJSInterop extends plugins.PluginComponent
         }
 
         val ownerLoadSpec = jsInterop.jsNativeLoadSpecOf(sym.owner)
-        val jsName = jsInterop.jsNameOf(sym)
+        val JSName.Literal(jsName) = jsInterop.jsNameOf(sym)
```

Comment:
Yes but that would be terribly difficult to jungle with wrt. backward binary compat of the scalac Symbols at this point.

I'm looking in the direction of using a scalac `InfoTransform`. It might save the situation.